odoo/http.py

```diff
@@ -703,29 +705,44 @@ def vacuum(self):
                     os.unlink(path)
 
 
-class Session(dict):
+class Session(collections.UserDict):
```

Comment:
Or keep inheriting from `werkzeug.session` :roll_eyes:

Reply:
> Or keep inheriting from `werkzeug.session` so we don't re-invent the wheel roll_eyes

That as well but

>  (its FP/AL's idea to reimplement our own dirty flag on top of dict, instead of just using werkzeug.session)

 :shrug: 

Although (in fairness) it's not really part of werkzeug anymore, we vendored it since it was dropped from werkzeug 1.0 (and moved to a separate package which was not available on some distros).